code/controllers/Processes/supply.dm

```diff
@@ -100,13 +100,16 @@ var/datum/controller/supply/supply_controller = new()
 		EC.name = "\proper[MA.name]"
 		EC.value = 0
 		EC.contents = list()
+		var/base_value = 0
 
 		// Must be in a crate!
 		if(istype(MA,/obj/structure/closet/crate))
 			var/obj/structure/closet/crate/CR = MA
 			callHook("sell_crate", list(CR, area_shuttle))
 
 			points += CR.points_per_crate
+			if(CR.points_per_crate)
+				base_value = CR.points_per_crate
```

Comment:
Editing EC.value directly with `EC.value += CR.points_per_crate` makes the `base_value` var unnecessary

Reply:
CR is only defined in that IF statement and if the MA is a crate in the first place, and EC.value has to only be edited after the whole ifelse statement is done and points are assigned.

Reply:
I see. Nevermind, then.